LICENSE-AGPL3.txt

```diff
@@ -174,7 +174,7 @@ measures.
 
   When you convey a covered work, you waive any legal power to forbid
 circumvention of technological measures to the extent such circumvention
-is effected by exercising rights under this License with respect to
+is affected by exercising rights under this License with respect to
```

Comment:
Changing the licensing for grammar isn't okay. Effected is the correct term here, regardless.